shogun-lib/src/main/java/de/terrestris/shogun/lib/service/BaseService.java

```diff
@@ -56,6 +58,24 @@
         return repository.findById(id);
     }
 
+    @PreAuthorize("hasRole('ROLE_ADMIN') or hasPermission(#entity, 'READ')")
```

Comment:
if the construct still exists, i would like to use something more generic:
`@PreAuthorize("hasRole(@configHolder.getSuperAdminRoleName())")`

Reply:
Currently the `@configHolder.getSuperAdminRoleName()` doesn't exist (yet), but I agree that we should reimplement it! Good catch.